src/images/svg/path.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-labelledby="path-icon-title">
+    <!-- Generator: Sketch 56.3 (81716) - https://sketch.com -->
+    <title id="path-icon-title">Path icon</title>
```

Comment:
Rename this to chevron? since it's for the  downward facing `>`

Reply:
Done